modules/geo-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -11,9 +11,6 @@ function validateGeometryAttributes(attributes) {
   if (!hasColorAttribute) {
     attributes.colors = {constant: true, value: new Float32Array([1, 1, 1])};
   }
-  if (!attributes.uvRegions) {
```

Comment:
Any reason to still leave this in, in case the #define is turned on?

Reply:
I don't see. If don't have uvRegions in the tile content there will be define and this attribute won't be used.